src/App.js

```diff
@@ -1,7 +1,79 @@
 import "./App.css";
+import { Component } from "react";
+import Cart from "./components/Cart";
+import CheckoutForm from "./components/CheckoutForm";
+import ProductCard from "./components/ProductCard";
+import productData from "./data/productData";
+import formatPrice from "./helpers/formatPrice";
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      allProductData: productData,
+      cartList: [],
+      subtotal: 0,
+      tax: 0,
+      total: 0
+    }
+  }
+
+  handleAddToCart = (product) => {
+    this.setState({
+      cartList: [...this.state.cartList, product],
+      subtotal: this.state.subtotal + product.price,
+      tax: this.state.tax + (product.price * 0.05),
+      total: this.state.total + product.price + (product.price * 0.05)
+    })
+  }
+
+  render() {
+    let productArr = this.state.allProductData.map(product => {
+      return (
+        <ProductCard 
+          key = {product.id}
+          product = {product}
+          handleAddToCart = {this.handleAddToCart}
+        />
+      )
+    });
+
+    let cartArr = this.state.cartList.map((item, i)=> {
+      return (
+        <Cart 
+          key = {i + '-' + item}
+          item = {item}
+        />
+      )
+    })
```

Comment:
```suggestion
    let cartArr = this.state.cartList.map((item, i)=> {
      return (
       <li>{item.name}: {formatPrice(item.price)}</li>
      )
    })
```
Like i mentioned in our 1:1 code review, it makes more sense to return one `<li>` for each product added rather than an entire Cart component. The Cart component should be the collection of all these `<li>`'s, plus the header "Cart", plus the Subtotal, Tax, and Total display.